core/modules/system/js/token.js

```diff
@@ -5,77 +5,321 @@
 
 "use strict";
 
-Backdrop.behaviors.tokenTree = {
+Backdrop.behaviors.tokenBrowser = {
   attach: function (context, settings) {
-    $(context).find('table.token-tree').once('token-tree', function () {
-      $(this).treetable({
-        'expandable': true,
-        'clickableNodeNames': true,
-        'stringCollapse': Backdrop.t('Collapse'),
-        'stringExpand': Backdrop.t('Expand')
-      });
-    });
-  }
-};
+    var ANCESTORS = {};
 
-Backdrop.behaviors.tokenInsert = {
-  attach: function (context, settings) {
-    // Keep track of which textfield was last selected/focused.
-    $(context).find('textarea, input[type="text"]').on('focus', function() {
-      Backdrop.settings.tokenFocusedField = this;
-    });
+    var buffer;
+    var tr_template;
+    var button_template;
+    var link_template;
+    var no_link_template;
+    var name_template;
+    var raw_template;
+    var description_template;
```

Comment:
The rest of this file uses camel case, could you adjust these variables here and throughout?

```suggestion
    var buffer;
    var trTemplate;
    var buttonTemplate;
    var linkTemplate;
    var noLinkTemplate;
    var nameTemplate;
    var rawTemplate;
    var descriptionTemplate;
```

Reply:
Done